ocaml/idl/datamodel.ml

```diff
@@ -3902,8 +3902,11 @@ let task =
 
 (** Many of the objects need to record IO bandwidth *)
 let iobandwidth =
-  [ field ~persist:false ~qualifier:DynamicRO ~ty:Float "read_kbs" "Read bandwidth (KiB/s)";
-    field ~persist:false ~qualifier:DynamicRO ~ty:Float "write_kbs" "Write bandwidth (KiB/s)" ]
+  [ field ~persist:false ~qualifier:DynamicRO ~ty:Float "read_kbs" "Read bandwidth (KiB/s)"
+      ~lifecycle:[ Removed, rel_ely, "Disabled in favour of RRDs"]
```

Comment:
I think we can "backdate" this to `rel_tampa` in which the fields definitely stopped being populated. Before, since `rel_orlando` (or somewhere around there), there was still an option to have the field set, which was disabled by default.

Same comment for all the other fields.